### ISSUE TITLE
[1.18.2 Forge] Crash when compatibilityMode is set to true

### ISSUE THREAD

**Issue by MoreThanHidden** [Leximon/Fluidlogged#6]
Blue Power Dev here, had an Issue logged due to rendering crashes took me a while to figure out which mod was causing the issues, seems to be Fluidlogged mod, assuming it is modifying the microblocks because they are water-loggable, which is breaking the renderer.

Open issues on Blue Power and JEI:
https://github.com/Qmunity/BluePower/issues/614 and https://github.com/mezz/JustEnoughItems/issues/3059

Had recommended setting the following in fluidlogged-common.toml:
```toml
#If compatibility mode is true, this mod will only apply to vanilla waterloggable block types(but will still work on modded blocks of those types).
compatibilityMode = true
```

Attempting to set compatibilityMode to true causes a crash...
[crash-2023-02-05_08.22.01-client.txt](https://github.com/Leximon/Fluidlogged/files/10609937/crash-2023-02-05_08.22.01-client.txt)
[latest.log](https://github.com/Leximon/Fluidlogged/files/10609940/latest.log)

**Comment by Leximon** [comment]
The forge version is unfortunately not maintained by the original port developer anymore. I might take a look again at this issue when I create a new forge version but as of now, I have to let this issue open.
Thanks for the report anyways!

**Comment by MoreThanHidden** [comment]
Fixed in bluepower-1.18.2-0.6.314-alpha on my side, but the crash when compatibilityMode != false is still an issue.

**Comment by Leximon** [comment]
It seems like the crash is only caused by this mod's old (not yet deprecated) version.
